lib/active_model/serializer.rb

```diff
@@ -17,32 +19,10 @@ class Serializer
     include Associations
     include Attributes
     include Caching
+    include Links
+    include Type
```

Comment:
we should just have a reference conversation we write about extracting to modules vs. classes to link to when this comes up :)